content/fr/docs/4.directory-structure/9.pages.md

```diff
@@ -244,6 +244,18 @@ export default {
 Pour en savoir plus sur la propriété d'observation, consultez notre chapitre [Data Fetching](/docs/features/data-fetching) chapter
 ::
 
+### key
+
+Comme la propriété `key` qui peut être utilisée sur les composants Vue dans les modèles comme une indication pour le DOM virtuel, cette propriété permet à la valeur de la clé d'être définie à partir de la page elle-même (plutôt que le composant parent).
+
+Par défaut dans Nuxt, cette valeur sera le `$route.path`, ce qui signifie que naviguer vers une route différente assurera la création d'un composant de page propre. Logiquement équivalent à :
+
+```html
+<router-view :key="$route.path" />
+```
+
+La propriété peut être une `String` ou une `Function` qui prend la route comme premier argument.
+
```

Comment:
Same here, I had this part translated by a translation service. I'm not sure if the translation is correct or not, so I would like you to check it for me. 🙏

Reply:
```md
### key

Comme la propriété `key` qui peut être utilisée sur les composants Vue dans les modèles comme une indication pour le DOM virtuel, cette propriété permet à la valeur de la clé d'être définie à partir de la page elle-même (plutôt que le composant parent).

Par défaut dans Nuxt, cette valeur sera `$route.path`, ce qui signifie que naviguer vers une route différente assurera la création d'un composant de page propre. Logiquement équivalent à :

```html
<router-view :key="$route.path" />
```

La propriété peut être une `String` ou une `Function` qui prend la route comme premier argument.
```